<launch>
    
    <param name="scan_line" type="int" value="16" />

    <!-- if 1, do mapping 10 Hz, if 2, do mapping 5 Hz. Suggest to use 1, it will adjust frequence automaticlly -->
    <param name="mapping_skip_frame" type="int" value="1" />

    <!-- remove too closed points -->
    <param name="LidarToPointsThres" type="double" value="0.1"/>

    <param name="mapping_line_resolution" type="double" value="0.2"/>
    <param name="mapping_plane_resolution" type="double" value="0.4"/>

    <param name="mapviz_filter_size" type="double" value="0.1"/>

    <!--  -->
    <param name="lidar_type" type="string" value="VLP16"/> 
    <remap from="/velodyne_points" to="/velodyne_points"/>

    <!-- nodes -->
    <node pkg="aloam" type="ascanRegistration" name="ascanRegistration" output="screen" />
    <node pkg="aloam" type="alaserMapping" name="alaserMapping" output="screen" />
        <param name="result_dir1" type="string" value="/home/multipleye/Dataset/201014_skt_lobby_day_lidar/aloam_mapping_highfrequency_pose.txt" />
        <param name="result_dir2" type="string" value="/home/multipleye/Dataset/201014_skt_lobby_day_lidar/aloam_mapping_pose.txt" />

    <!-- visulaization -->
    <arg name="rviz" default="true" />
    <group if="$(arg rviz)">
        <node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz" args="-d $(find aloam)/rviz/aloam.rviz" />
    </group>

</launch>


